extendr-api/src/functions.rs

```diff
@@ -131,7 +132,7 @@ pub fn new_env(parent: Environment, hash: bool, capacity: i32) -> Environment {
         .unwrap()
 }
 
-/// The base environment; formerly R_NilValue
+/// The base environment; formerly `R_NilValue`
```

Comment:
Is this formerly or formally?

Reply:
You know what? I think it is formerly. And _actually_ I think something weird happens when you use `NilValue` as environmnet.